src/installation/index.md

```diff
@@ -1,61 +1,167 @@
-# Rust on ESP targets
+# Setting Up a Development Environment
 
 With an understanding of the ecosystem surrounding Rust on Espressif chips, we can move on to actual development. If you are not aware of the two possible development approaches or do not understand the differences between writing `std` and `no_std` applications, please first read the [Ecosystem Overview] chapter.
 
-Let's take a moment to discuss the Rust support for the architectures of the Espressif chips in more detail. At this moment, Espressif SoCs are based on two different architectures: `RISC-V` and `Xtensa`. The support for those two architectures in the Rust programming language is very different.
+Let's take a moment to discuss the Rust support for the different architectures of the Espressif chips and the different approaches in more detail. At this moment, Espressif SoCs are based on two different architectures: `RISC-V` and `Xtensa`. The support for those two architectures in the Rust programming language is very different.
 
-## Rust in RISC-V targets
+[Ecosystem Overview]: ../overview/index.md
+
+## RISC-V targets
+
+The `RISC-V` architecture has support in the mainline Rust compiler so, the setup is relatively simple. There are two ways of proceeding with the installation:
+- Using [`espup`, a tool that will be covered later]
+- Using the official Rust tools
+
+If you only want to use `RISC-V` targets, you can use the official Rust tools, for this approach we need [`rustup`] installed, and a [Rust nightly toolchain] with the `rust-src` [component]. We can install a nightly toolchain with the `rust-src` component via:
+
+```bash
+rustup toolchain install nightly --component rust-src
+```
+
+This are the two recommended targets for most Espressif `RISC-V` chips:
+- For bare-metal (`no_std`) applications: `riscv32imc-unknown-none-elf`
```

Comment:
I personally still feel like we should be installing the imac target by default, but we should discuss this in one of our roadmap meetings.

Reply:
Updated it to `riscv32imac-unknown-none-elf`

Reply:
Really really sorry, but can you change it back :laughing:, I just did a couple of PRs to make imc the default for C3 and C2, see https://github.com/esp-rs/esp-template/pull/51 and https://github.com/esp-rs/esp-hal/pull/370